index.bs

```diff
@@ -269,39 +269,47 @@ Report long tasks {#report-long-tasks}
 
         For each <a>top-level browsing context</a> |topmostBC| in |top-level browsing contexts|:
 
-        1. Add |topmostBC|'s Window's <a>relevant Realm</a> to |destinationRealms|.
+        1. Add |topmostBC|'s <a>active document</a>'s <a>relevant Realm</a> to |destinationRealms|.
         1. Let |descendantBCs| be |topmostBC|'s <a>active document</a>'s <a>list of the descendant browsing contexts</a>.
-        1. For each |descendantBC| in |descendantBCs|, add |descendantBC|'s Window's <a>relevant Realm</a> to |destinationRealms|.
+        1. For each |descendantBC| in |descendantBCs|, add |descendantBC|'s <a>active document</a>'s <a>relevant Realm</a> to |destinationRealms|.
+    
+    1. A user agent may remove some <a>JavaScript Realms</a> from |destinationRealms|.
+
+    Note: this removal could be used to avoid reporting long tasks for <a>JavaScript Realms</a> that the user agent handles in a separate process. However, this concept is not specified precisely.
+
+    Issue(75): there is some ongoing discussion regarding the scope of which {{Document|Documents}} gain visibility over which long tasks, so this logic could change in the future.
 
     1. For each |destinationRealm| in |destinationRealms|:
 
         1. Let |name| be the empty string. This will be used to report <a>minimal culprit attribution</a>, below.
         1. Let |culpritSettings| be <code>null</code>.
-        1. If the |task| argument was not provided, set |name| to "<code><a>unknown</a></code>".
-        1. Otherwise: process |task|'s <a>script evaluation environment settings object set</a> to determine |name| and |culpritSettings| as follows:
+        1. Process |task|'s <a>script evaluation environment settings object set</a> to determine |name| and |culpritSettings| as follows:
 
             1. If |task|'s <a>script evaluation environment settings object set</a> is empty: set |name| to "<code><a>unknown</a></code>" and |culpritSettings| to <code>null</code>.
-            1. If |task|'s <a>script evaluation environment settings object set</a>'s length is greater than one: set |name| to "<code><a>multiple-contexts</a></code>" and |culpritSettings| to <code>null</code>.
-            1. If |task|'s <a>script evaluation environment settings object set</a>'s length is one:
+            1. Otherwise, if |task|'s <a>script evaluation environment settings object set</a>'s length is greater than one: set |name| to "<code><a>multiple-contexts</a></code>" and |culpritSettings| to <code>null</code>.
+            1. Otherwise, i.e. if |task|'s <a>script evaluation environment settings object set</a>'s length is one:
                 1. Set |culpritSettings| to the single item in |task|'s <a>script evaluation environment settings object set</a>.
-                1. Let |destinationSettings| be |destinationRealm|'s <a>relevant settings object</a>.
+                1. Let |destinationSettings| be |destinationRealm|'s <a for=Realm>settings object</a>.
```

Comment:
FYI this could be `[=Realm/settings object=]`. (Same for other `for=""` uses.)

Reply:
Replaced all `<a></a>`s with `[==]`s in this algorithm since you seem to like them more and I don't have a strong preference :)